src/Security/GitHubAuthenticator.php

```diff
@@ -79,12 +79,14 @@ public function authenticate(Request $request): Passport
                     return $existingUser;
                 }
 
-                if ($userRepo->findOneBy(['usernameCanonical' => mb_strtolower($ghUser->getNickname())])) {
-                    throw new AccountUsernameExistsWithoutGitHubException($ghUser->getNickname());
+                $nickname = $ghUser->getNickname();
```

Comment:
Here too a rebase should help some

Reply:
nope, no change here

![Bildschirmfoto von 2022-01-30 16-59-02](https://user-images.githubusercontent.com/2852185/151707123-164fdd57-700e-49f9-bf85-4b47d53088db.png)

do i miss sth?

Reply:
No this didn't change but the githubauthenticator I already stored nickname somewhere, but maybe it's some other place sorry

Reply:
ah, you're right. yup, missed that. moved the check in 20e1a196f58ace847bb1d6028d3803bcfe7303ce